src/style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  margin: 0;
+}
```

Comment:
checklist

Reply:
I understood wrongly the last requirment of the task.